frontend/packages/noobaa-storage-plugin/src/components/health-card/health-card.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { HealthBody, HealthItem } from '@console/internal/components/dashboard/health-card';
+import { HealthState } from '@console/internal/components/dashboard/health-card/states';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
please add empty line between `@console/internal/module/k8s` and `../../models` imports